sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNodeAddress.java

```diff
@@ -128,15 +128,11 @@ public boolean isTransportSecurity() {
      * @return                          the insecure managed node address
      */
     public ManagedNodeAddress toInsecure() {
-        var port = this.port;
-
-        switch (this.port) {
-            case PORT_NODE_TLS:
-                port = PORT_NODE_PLAIN;
-                break;
-            case PORT_MIRROR_TLS:
-                port = PORT_MIRROR_PLAIN;
-        }
+        var port = switch (this.port) {
```

Comment:
_IIRC_ We cannot use pattern matching switch statements as that is a new Java features. Although, I do find this odd now considering we do compile with `release = 8`.

Reply:
How do we confirm this?